site/sfguides/src/servicenow_to_snowflake_connector/servicenow_to_snowflake_connector.md

```diff
@@ -0,0 +1,364 @@
+
+### On the ServiceNow Other hand
+Duration: 4
+
+1. Log on to your ServiceNow developer instance.
+1. From the main page, select **All** and search **Application Registry**.
+
+![Application Registry](assets/now_reg_auth.png)
+1. Select **New** in the upper right-hand side of the window.
+1. Select **Create an OAuth API endpoint for external clients**. 
+1. Give the endpoint a name, such as **Snowflake_connector**. Leave the client secret blank. This will autofill.
+1. Paste in the redirect URL that was generated on the Snowflake hand. 
+
+![Oauth](assets/now_oauth_endpoint.png)
+1. Select **Submit**. The window closes.
+1. Select the registry you just created to re-open it. 
+1. Note that the **Client id** and **Client secret** are auto-generated. 
+1. Copy the **Client id**.
+
+Now, time to jump back to the Snowflake configuration tab.
+
+### Now Let's Shake
+Duration: 1
+
+1. Paste the  **Client id** from ServiceNow into the Snowflake configure pop-up.
+1. Go back to the ServiceNow tab and copy the **Client secret** and paste it into the Snowflake configure pop-up. 
+1. No need to change the Advanced Settings, but feel free to check them out.
+ ![Connect](assets/now_connect.png)
+1. Select **Connect**. Your ServiceNow accounts pops up and requests to connect to Snowflake. 
+![check](assets/now_check.png)
+1. Select **Allow**.
+The connection is established between the two systems. 
+
+To verify the connection, select the three dots [...] and **View Details**. At the top of the pop-up you will see the date **ServiceNow** Authenticated.
+
+![authenticated](assets/authenticated.png)
+
+Select **Done**. 
+
+> aside negative
+> If you are having issues, perhaps the password wasn't copied. Unlock the password field and copy and paste the text.
+## Configure the Connector
+Duration: 5
+
+Under the status for the connector, which displays "Choose Resources", select **Configure**.
+
+This displays the Configure Connector dialog. By default, the fields are set to the names of objects that are created when you configure the connector.
+
+![default config](assets/configuredefaults.png)
+
+Check out [Configuring the Snowflake Connector for ServiceNow](https://other-docs.snowflake.com/en/connectors/servicenow/servicenow-installing-ui.html#configuring-the-snowflake-connector-for-servicenow) for more information on these fields. 
+
+Select **Configure**. The dialog box closes and the status of the connector changes to Provisioning. It can take a few minutes for the configuration process to complete.
+
+> aside negative
+> Watch out!!! The created warehouse is created as a **Large** and with a auto timeout of 10 minutes. So this means, if you set to refresh every hour, the Large warehouse (8 credits/hour) will wake up for a minimum of 10 minutes every hour.  For this lab, you don't need all the power! Go to Admin-> Warehouses -> SERVICENOW_WAREHOUSE -> ... > Edit, and change this to an XSMALL, and the auto timeout to one minute.
```

Comment:
A warehouse can also be created beforehand and provided here, it won't be recreated as LARGE

Reply:
trying to follow the automatic parts as much as possible.